www/perl-lib/SMT/NCCRegTools.pm

```diff
@@ -103,8 +103,30 @@ sub new
     $self->{URI}      = $ruri;
     $self->{AUTHUSER} = $user;
     $self->{AUTHPASS} = $pass;
-    bless($self);
 
+    my $regsharing = SMT::Utils::hasRegSharing();
+    if (! $regsharing) {
+        my $msg = 'Could not read SMT configuration file';
+        printLog($self->{LOG}, $self->vblevel(), LOG_WARN, __($msg));
+        $regsharing = 'false';
+    }
+    elsif ($regsharing eq 'true') {
```

Comment:
hasRegSharing() does not return the string 'true'. So this will not work. simply use only "else"